src/link.cpp

```diff
@@ -1792,28 +1792,12 @@ static void construct_linker_job_elf(LinkJob *lj) {
         if (g->libc != nullptr) {
             if (!g->have_dynamic_link) {
                 lj->args.append("--start-group");
-                if (!target_is_android(g->zig_target)) {
-                    lj->args.append("-lgcc");
-                    lj->args.append("-lgcc_eh");
-                }
```

Comment:
Just a note to anyone else looking at this: I approved this change, it's time to lose our training wheels and rely on libcompiler_rt unconditionally. Here's to dropping support now for `-lgcc` and never looking back. :wine_glass:

Reply:
gcc is still more portable than compiler-rt here.

Reply:
@krytarowski this is in master branch now; if you have some reasoning why `-lgcc` is ever preferable to [zig's compiler-rt](https://ziglang.org/download/0.5.0/release-notes.html#compiler-rt), I invite you to share your knowledge in a new issue

Reply:
Do you mean that this is a builtin version from zig? If so, then this is fine!